IGC/AdaptorOCL/SPIRV/SPIRVReader.cpp

```diff
@@ -1576,7 +1576,11 @@ void SPIRVToLLVMDbgTran::transDbgInfo(SPIRVValue *SV, Value *V) {
             Line->getColumn(), scope, iat);
 
         if(scope && !isa<DIFile>(scope))
+#if LLVM_VERSION_MAJOR >= 12
+            I->setDebugLoc(DILocation::get(scope->getContext(), Line->getLine(), Line->getColumn(),
+#else
             I->setDebugLoc(DebugLoc::get(Line->getLine(), Line->getColumn(),
+#endif
```

Comment:
I haven't looked at when `DILocation::get()` was introduced. I will do that tomorrow.

Reply:
What's the earliest version of LLVM that IGC supports?

Reply:
Nevermind. I reviewed git changes back to about LLVM 6 or 7 and no substantial changes went into `class DILocation` or `class MDNode`, so `DILocation::get()` should work for those versions. I couldn't confirm it by actually building IGC because I don't have any machine using such an old LLVM version.